Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    apivore (1.0.0)
+    apivore (1.1.0)
```

Comment:
Good point, the reasoning makes sense -- PR to do this: https://github.com/westfieldlabs/apivore/pull/51